package.json

```diff
@@ -29,8 +29,7 @@
     "key-mirror": "1.0.1",
     "localforage": "1.7.3",
     "localforage-observable": "1.4.0",
-    "mark.js": "8.11.1",
```

Comment:
Looks like this line was accidentally removed

Reply:
Oh thanks, I thought all are good after I fixed merge conflicts yesterday.  Done reverting this line.